index.js

```diff
@@ -103,9 +103,9 @@ module.exports = class Autobase extends EventEmitter {
   }
 
   _onOutputTruncated (output, length, forkId) {
-    for (const view of this._views) {
-      view._onOutputTruncated(output, length, forkId)
-    }
+    if(!this.view) return
```

Comment:
Formatting issue there (should be `if (!this.view)`). If you run `npm t` in the module it should run standard first for formatting.

Reply:
ok il will format that (i work with prettier usually.)